README.md

```diff
@@ -1200,31 +1200,58 @@ Translations of the guide are available in the following languages:
     end
     ```
 
-  * <a name="no-and-or-or"></a>
-    The `and` and `or` keywords are banned. The minimal added readability is just
-    not worth the high probability of introducing subtle bugs. For boolean
-    expressions, always use `&&` and `||` instead. For flow control, use
-    `if` and `unless`; `&&` and `||` are also acceptable but less clear.
-    <sup>[[link](#no-and-or-or)]</sup>
+  * <a name="no-and-or-or"></a><a name="and-or-flow"></a>
+    Distinguish between `&&`/`||` logical operators and `and`/`or` control flow
```

Comment:
I'd phrase this more like a rune - e.g. Don't use `and` and `or` as logical operators...